internal/cli/iam/teams/create.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package teams
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+var createTemplate = "Team '{{.Name}}' created.\n"
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	name      string
+	userNames []string
+	store     store.TeamCreator
+}
+
+func (opts *CreateOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	r, err := opts.store.CreateTeam(opts.ConfigOrgID(), opts.newTeam())
+	if err != nil {
+		return err
+	}
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newTeam() *atlas.Team {
+	return &atlas.Team{
+		Name:      opts.name,
+		Usernames: opts.userNames,
+	}
+}
+
+// mongocli iam team(s) create <name> [--orgId orgId]
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: createTeam,
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunEOrg(
+				opts.init,
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.name = args[0]
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringSliceVar(&opts.userNames, flag.Username, []string{}, usage.Username)
```

Comment:
can we have a different `usage.Username` to explain this is a list of members of the team?